plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestHiveGlueMetastore.java

```diff
@@ -1453,7 +1454,7 @@ private static PartitionValues make(String... values)
 
         private PartitionValues(List<String> values)
         {
-            this.values = ImmutableList.copyOf(requireNonNull(values, "values is null"));
+            this.values = unmodifiableList(requireNonNull(values, "values is null"));
```

Comment:
> The value is nullable.

How come? You have `requireNonNull`.

Reply:
I mean the element, not `values` itself.

Reply:
sorry, wasn't aware of this PR and posted https://github.com/trinodb/trino/pull/14799 as a streamlined verison of https://github.com/trinodb/trino/pull/14796 that @findinpath pinged me about

note that this change logically partially reverts bba041c4e0fc4d07da07abfe079dc11849e84218  -- the `PartitionValues`  becomes mutable again.
You need a defensive copy of the constructor arg _and_ wrapping with `unmodifiableList`